specs/Daito/Log_specification.md

```diff
@@ -0,0 +1,52 @@
+# Log specification
+**Authors:** [liuqinguestc](https://github.com/liuqinguestc)
+
+## Summary
+
+OpenSDS log is not uniform and standardized, and the log format is various.We need to develop a unified specification log format to facilitate location and troubleshooting.
+
+## Motivation
+
+Each OpenSDS developer uses a different log format, and the log location is different, causing problems for positioning and troubleshooting.We need to standardize the log format so that developers and testers can quickly locate problems, solve problems, improve development efficiency, and reduce communication costs.
```

Comment:
Please make sure there is a space after each period before another sentence.

s/positioning/locating the logs